service/controller/v11/key/key.go

```diff
@@ -295,6 +295,16 @@ func Region(customObject v1alpha1.AWSConfig) string {
 	return customObject.Spec.AWS.Region
 }
 
+func RegionARN(customObject v1alpha1.AWSConfig) string {
+	regionARN := "aws"
+
+	if customObject.Spec.AWS.Region == "cn-north-1" {
```

Comment:
Is this the only cn region? Will we have to do changes if Adidas wants to use another cn region?

Reply:
We are only using this one now, there's also `cn-northwest-1` but it is not being used. In general the approach for region-dependent data is to only include the regions being used, see for instance how we determine the AMIs to be used by region https://github.com/giantswarm/aws-operator/blob/master/service/controller/v11/key/key.go#L391-L397

Reply:
Would be good to use `key.Region` here. As we will restructure the cluster CRs at some point.

Reply:
Good point! Done.